guide/glossary.md

```diff
@@ -251,19 +251,37 @@ A standard for multi-signature that uses Schnorr signatures. Previously, the mor
 
 ### Node
 
-Node refers to [software]({{ '/guide/getting-started/software/#nodes' | relative_url }}) that participates in the bitcoin network. It exchanges transaction data with other nodes, stores some or all of it, and verifies that transactions are valid. There is also dedicated [node hardware]({{ '/guide/getting-started/hardware/#hardware-wallets' | relative_url }}).
+A node refers to a Bitcoin client that participates in the Bitcoin network.
```

Comment:
TBH, I prefer the original text. find the term "Bitcoin client" confusing (software is more direct) and think that explaining the term "participates" is important.

Reply:
I've mostly been using `application` to refer to software, as the term `app` has become so mainstream.

Reply:
it relates to the Bitcoin Client entry earlier on in the glossary. Application sounds too broad and encompasses software that does not connect to and/or run the Bitcoin network.